hms-app/src/components/lists/AllHackathonList.tsx

```diff
@@ -21,6 +21,10 @@ const useStyles = createStyles((theme) => ({
   },
 }))
 
+enum Enum {
+  FullInfo = 'FULLINFO',
```

Comment:
Maybe easier to read
```suggestion
  FullInfo = 'FULL_INFO',
```